sklearn/metrics/scorer.py

```diff
@@ -480,16 +480,13 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 # Standard Classification Scores
 accuracy_scorer = make_scorer(accuracy_score)
-f1_scorer = make_scorer(f1_score)
 balanced_accuracy_scorer = make_scorer(balanced_accuracy_score)
 
 # Score functions that need decision values
 roc_auc_scorer = make_scorer(roc_auc_score, greater_is_better=True,
                              needs_threshold=True)
 average_precision_scorer = make_scorer(average_precision_score,
                                        needs_threshold=True)
-precision_scorer = make_scorer(precision_score)
-recall_scorer = make_scorer(recall_score)
```

Comment:
I would rather add them to SCORERS as they are legit scorers

Reply:
This [commit](https://github.com/scikit-learn/scikit-learn/commit/081a5546988e4514afecf00250901f66e29b7543#diff-2ed8e107a97b6a01487cca491ae1360cL345) has removed them in favor of ['macro', 'micro', 'samples', 'weighted'] versions